### PR TITLE
doc: Release instructions: Clarify that Java versions have no leading…

### PR DESCRIPTION
**Pull request by michael-berlin**
… "v".

### CODE REVIEW

**Comment by alainjobart** [comment]
LGTM

File: doc/internal/ReleaseInstructions.md [hunk 1]
```diff
@@ -275,6 +276,7 @@ git checkout master
 git pull --ff-only upstream master
 
 # Example:
```
Comment (ashudeep-sharma) [inline doc/internal/ReleaseInstructions.md]: There is no Example, should we add something after _Example:_ as well?

Comment (michael-berlin) [inline doc/internal/ReleaseInstructions.md]: `JAVA_VERSION=2.1.1-SNAPSHOT` on the next line is meant as the example.

You can also see that in the rendered version: http://vitess.io/internal/release-instructions.html#bump-java-snapshot-version

Do you think that's not clear enough?

Comment (ashudeep-sharma) [inline doc/internal/ReleaseInstructions.md]: I was comparing it with the Example in line 212, where you have given an example on the same line where Example: is mentioned.

**Pull request merged**